cmds/core/mktemp/mktemp.go

```diff
@@ -30,14 +30,21 @@ package main
 
 import (
 	"fmt"
+	"io"
 	"log"
 	"os"
 	"strings"
 
 	flag "github.com/spf13/pflag"
 )
 
-type mktempflags struct {
+type cmd struct {
+	stdout io.Writer
+	flags  flags
+	args   []string
+}
+
+type flags struct {
```

Comment:
can you consider name this after sth else like `Option` ?  `flags` is almost like standard pkg `flag`

Reply:
It should be something related to flags, maybe cmdFlags or something. Options can be confusing as well, people may think that is something related to WithXXX option pattern.

Anyway it kind of experiment of moving away from globals. Not even sure if should do it at all.

Reply:
remember people use package names all the time for variables; you can find variables named path everywhere. I think this is descrptive.

Reply:
I'm gonna close this comment, I think it's ok.